strawberry_django/filters.py

```diff
@@ -118,7 +121,19 @@ def build_filter_kwargs(filters):
     return filter_kwargs, filter_methods
 
 
-def apply(filters, queryset, pk=UNSET):
+@functools.cache()
+def function_allow_passing_info(filter_method: FunctionType) -> bool:
+    argspec = inspect.getfullargspec(filter_method)
+
+    if "info" in getattr(argspec, "args", []) or "info" in getattr(
+        argspec, "kwargs", []
+    ):
+        return True
+    else:
+        return False
```

Comment:
:)

```suggestion
    return "info" in getattr(argspec, "args", []) or "info" in getattr(
        argspec, "kwargs", []
    )
```

Reply:
Holy shite never thought of that -- thanks! 😁